academics/ImportantDates.md

```diff
@@ -5,53 +5,34 @@ sidebar_label: Important Dates
 slug: /dates
 ---
 
```

Comment:
please add the last update date in the format found here https://github.com/uwindsorcss/wiki/blob/master/REQUIREMENTS.md#last-update-date